src/AcmLabels/AcmLabels.tsx

```diff
@@ -44,23 +41,47 @@ export function AcmLabels(props: {
 
     if (props.labels === undefined) return <Fragment></Fragment>
 
+    const [showMore, setShowMore] = useState(false)
+
+    /* istanbul ignore next */
+    const collapsedText = props.collapsedText ?? `${hidden.length} more`
+
+    /* istanbul ignore next */
+    const expandedText = props.expandedText ?? 'Show less'
+
+    const onClick = useCallback(() => setShowMore((showMore) => !showMore), [])
+
+    /* istanbul ignore next */
+    const onKeyPress = useCallback((e) => {
+        if (e.key === 'Enter' || e.key === ' ') {
+            setShowMore(!showMore)
+            e.preventDefault()
+        }
+    }, [])
```

Comment:
Is there difficulty in getting some test coverage on these lines?

Reply:
The first two are just replacing the text that is shown and are simple enough that creating tests to cover that does not really buy us anything.

The key press, I cannot figure out how to get the user input testing library to simulate the keypress. :(